lib/beaker/command.rb

```diff
@@ -148,9 +148,17 @@ def environment_string_for host, env
         env_array << "#{key.to_s.upcase}=\"#{val}\""
       end
 
-      environment_string = env_array.join(' ')
-
-      "env #{environment_string}"
+      if host['communicator'] =~ /bitvise/
```

Comment:
I think it might be easier to have something like an `is_cygwin?` method that does a check for one of the unix commands or the /cygdrive/c/ directory.

What do you think?